### DISCUSSION TITLE
Slider ticks disappearing when moving the thumb

### DISCUSSION THREAD

**Discussion by Frodo1980** [vuetifyjs/vuetify#14275] [Q&A]
Hi,

please have a look at [codesandbox](https://codesandbox.io/s/wandering-cache-quvgq?file=/src/App.vue)

For testing just fill in each required formfield a "1" and then confirm the form.

The first range-slider should give you the opportunity to select a range.
The second slider should give you this selected range in detail.
At 5 and 5,5% there are two ticks defined. And at the beginning and at the end of the range there are ticks too. When moving the thumbs of the range-slider these ticks disappear somehow. Moving slow brings them back (sometimes). Any ideas for this behaviour? I want them to be shown all the time.

Thanks for your hints

**Comment by ghost** [comment]
The problem is that you are updating the array of ticks every time the range-slider moves only a tiny bit, so if someone moves it from 5,50 to 6 the watch function for your ticksArray is being called 60 times. I'd recommend only renewing the array when the user has finished its selection as the calls are being made clientside. Which depending on the specs of the user can take quite a long time

**Reply by Frodo1980** [reply]
Thank you Nick, that sounds good.
